org-code-javabuilder/util/build.gradle

```diff
@@ -0,0 +1,22 @@
+/*
```

Comment:
Good call on `protocol`. I'll update that.

For your third note here. Any advice/feedback you can give would be welcome. I spent several hours trying to come up with a design pattern that would let us separate Neighborhood from Javabuilder and yet still allow both of them to communicate with the client. I couldn't come up with anything and set up a _very_ hacky workaround that will let us unblock curriculum while we do some in-depth design. https://github.com/code-dot-org/javabuilder/pull/36

Reply:
I'm a little stuck on this too.  I think we either need a "global" of some sort (similar to System.out) or we need to initialize World externally so that a handler can be injected.  Maybe ping me offline if you want to brainstorm some more on this?

Reply:
👍 Thanks for putting some thought into it. I'll get a design brainstorm meeting set up.